components/dashboard/src/AppNotifications.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useCallback, useEffect, useState } from "react";
+import Alert, { AlertType } from "./components/Alert";
+import { useFeatureFlag } from "./data/featureflag-query";
+
+const KEY_APP_DISMISSED_NOTIFICATIONS = "gitpod-app-notifications-dismissed";
+
+interface Notification {
+    id: string;
+    type: AlertType;
+    message: JSX.Element;
+}
+
+const SCHEDULED_DOWNTIME: Notification = {
+    id: "230924-scheduled-downtime",
+    type: "info",
+    message: (
+        <span className="text-md">
+            To improve the reliability of gitpod.io we are performing a scheduled maintenance this{" "}
+            <span className="font-semibold">Sunday, Sept. 24th, 05:00-06:00 UTC</span>.{" "}
```

Comment:
Would it make sense to translate this to the user's time instead of using UTC? We can still keep the UTC time by wrapping it in a [`<time>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).

Reply:
Heave +1 :)

Reply:
Done!